frontend/src/Login/authReducer.js

```diff
@@ -0,0 +1,14 @@
+import { LOGIN } from './actionCreator';
+
+const defaultState = {
+  isLogin: false,
```

Comment:
isLoggedIn? or isLoggingIn? which one is it? please rename, actually I can't decide what you wanted, the process, like isLoading, or the state of the user, who isLoggedIn. I guess it's the logged in status, right?

```suggestion
  isLoggedIn: false,
```